x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -27,8 +27,7 @@ import { useTestIdGenerator } from '../../../../hooks/use_test_id_generator';
 import { useDataTestSubj } from '../../hooks/state_selectors/use_data_test_subj';
 
 const CommandInputContainer = styled.div`
-  background-color: ${({ theme: { eui } }) => eui.euiColorGhost};
-  border-radius: ${({ theme: { eui } }) => eui.euiBorderRadius};
+  background-color: ${({ theme: { eui } }) => eui.euiFormBackgroundColor};
```

Comment:
using the backgroundColor is to ensure that it will be updated appropriately when the theme changes right?

Reply:
Right. The input area and side panel actually have a slight (very slight) difference in colors. We may need to adjust the colors further if Meril does not feel that using this Eui variable is enought, but I try hard to influence all to keep to the variables so that we don't have to think about theming